File: Practica-POO/Semana01/Ejercicio02/Circle.py
class Circle:
    def __init__(self,radius):
        self.radio = radius

    def CalculateArea(self):
        return 2*3.14*self.radio


circle1 = Circle(2)
area1 = circle1.CalculateArea()
print(f"El area es: {area1} metros")

File: Practica-POO/Semana01/Ejercicio02/Circle.cpp
#include <iostream>
using namespace std;

class Circle{
    public:
    double radius;

    Circle(double r){
        radius = r;
    }

    double CalculateArea(){
        return 3.14*(radius*radius);
    }
};

int main(){
    Circle circle1(2);

    double area1 = circle1.CalculateArea();

    cout << "El area es: " << area1 << endl;
}